class ArgumentError(Exception):
    pass

class SearchError(Exception):
    pass
